src/input/base.jsx

```diff
@@ -41,11 +41,15 @@ class TextInputComponent extends InputComponent {
     return this.getInput().selectionEnd || this.state.value.length;
   }
 
+  isDOMValueEqualTo(domValue, rawValue) {
+    return domValue === rawValue;
+  }
+
   // Fixes the autofill feature in some of the dumbass browsers that are incapable of
```

Comment:
Little unrelated but could be specified who those brwosers are? In case one day we wont support them anyway and can kick that code?

Reply:
Very hard to say because the comment is very old and the package already was deprecated